gcloud/bigtable/table.py

```diff
@@ -139,6 +139,36 @@ def create(self, initial_split_keys=None):
         # We expect a `._generated.bigtable_table_data_pb2.Table`
         client._table_stub.CreateTable(request_pb, client.timeout_seconds)
 
+    def rename(self, new_table_id):
+        """Rename this table.
+
+        .. note::
+
+            This cannot be used to move tables between clusters,
+            zones, or projects.
+
+        .. note::
+
+            The Bigtable Table Admin API currently returns
+
+                ``BigtableTableService.RenameTable is not yet implemented``
+
+            when this method is used. It's unclear when this method will
+            actually be supported by the API.
```

Comment:
Maybe, but even if it were intended, it doesn't sound like it's being helpful. Do the descriptions of the status codes used in these two circumstances better align with the experience of the developer on the receiving end of them?